base_business_document_import/components/edi_input_process_business_document.py

```diff
@@ -0,0 +1,163 @@
+# Copyright 2020 Creu Blanca
+# @author: Enric Tobella
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.addons.component.core import AbstractComponent
+
+
+class EDIInputProcessBusinessDocumentImport(AbstractComponent):
```

Comment:
This functions are taken from `business.document.import`. On future, they should need to be passed here :wink: